utils/node-js/compat.js

```diff
@@ -0,0 +1,51 @@
+const glob = require('glob');
+const fs = require('fs-extra');
+
+/**
+ * During the `skyux build-public-library` step, this function injects
+ * the contents of the locales resources file when referenced via
+ * a `require` statement in a source file.
+ * This is done to prevent a breaking change, since SkyResources fetches
+ * the resource string synchronously, and SkyAppResources asynchronously.
+ * Once a library can release a breaking change to adopt SkyAppResources,
+ * this file should no longer be used.
+ */
+function injectLocaleResourcesContents() {
+  const resourceFileExists = fs.pathExistsSync('./src/assets/locales/resources_en_US.json');
```

Comment:
Should we make the file path a constant, since its used a couple times in here?

Reply:
Where else is it used?